personalsite/src/App.js

```diff
@@ -0,0 +1,94 @@
+import './App.css';
+
+function App() {
+  
+  const buttonFunction = () => {
+    console.log("the button was clicked");
+    var x = document.getElementById("myDIV");
+    if (x.style.display === "none") {
+    x.style.display = "block";
+    }
+    else {
+    x.style.display = "none";
+    }
+  }
+
+  const swimming = {
+    name: "Swimming",
+    years: 10
+  }
+
+  const diving = {
+    name: "Diving",
+    years: 4 
+  }
+
+  const waterpolo = {
+    name: "Waterpolo",
+    years: 3 
+  }
+
+  const mySports = [swimming, diving, waterpolo];
+  const sportMap = mySports.map((sport) => {
+    return(
+    <div className = "sport">
+        <p>
+          {sport.name} for {sport.years} years.
+        </p>
+      </div>
+    )
+  })
+  
+
+ 
+  return (
+    <div className="App">
+    
+        <meta content="width=device-width, initial-scale=1"></meta>
+      <head>
```

Comment:
totally forgot to mention this, so theres no way youd know this but we dont use `<head>` and `<body>` tags in React html. You probably noticed that the head stuff wasnt working. if you want to change the title or favicon you'd do that in `public/index.html` (ill drop a comment there too so you'll see where it goes). The `<body>` can just be replaced with a `<div>`